integration/kube_integration_test.go

```diff
@@ -1213,13 +1213,11 @@ func tlsClientConfig(cfg *rest.Config) (*tls.Config, error) {
 		return nil, trace.BadParameter("failed to append certs from PEM")
 	}
 
-	tlsConfig := &tls.Config{
+	return &tls.Config{
 		RootCAs:      pool,
 		Certificates: []tls.Certificate{cert},
 		ClientAuth:   tls.RequireAndVerifyClientCert,
-	}
-	tlsConfig.BuildNameToCertificate()
```

Comment:
Call to `BuildNameToCertificate()` was remove here, but left here? https://github.com/gravitational/teleport/pull/16507/files#diff-cdfe0b66f7090f9a5e0759d6cce323f7dde1ee1f0f49402d6a7652cd91b69430R2012